variables.tf

```diff
@@ -259,6 +259,11 @@ variable "ssh_agent" {
   default     = false
 }
 
+variable "connection_timeout" {
+  description = "Timeout in minutes for SSH connections"
+  default     = 30
```

Comment:
It depends if your vm health status is WARNING it may take some time to connect.. It takes around 10-13 mins as per my tests. If the vm status is ACTIVE/OK then it is quick.